src/cartesian/Area.tsx

```diff
@@ -463,6 +463,7 @@ class AreaWithState extends PureComponent<Props, State> {
             data={this.props.data}
             xAxisId={xAxisId}
             yAxisId={yAxisId}
+            zAxisId={0}
```

Comment:
does is matter that these don't care about z-axis? Why can't we just leave it undefined?

Reply:
Could be undefined too. I used zero so that I don't have to change the type to `string | number | undefined`.

Reply:
sounds good, as long as it doesn't cause issues when someone adds a ZAxis to a component that doesn't accept a ZAxis

Reply:
I can imagine we would want to add it later. It could be represented with a color or perhaps a thickness of a line. Until then it will be ignored.